src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,1327 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+        |   |   `--LITERAL_VOID -> void [12:11]
+        |   |--IDENT -> foo [12:16]
+        |   |--LPAREN -> ( [12:19]
+        |   |--PARAMETERS -> PARAMETERS [12:20]
+        |   |--RPAREN -> ) [12:20]
+        |   `--SLIST -> { [12:22]
+        |       |--VARIABLE_DEF -> VARIABLE_DEF [13:8]
+        |       |   |--MODIFIERS -> MODIFIERS [13:8]
+        |       |   |--TYPE -> TYPE [13:8]
+        |       |   |   `--LITERAL_INT -> int [13:8]
+        |       |   |--IDENT -> i [13:12]
+        |       |   `--ASSIGN -> = [13:14]
+        |       |       `--EXPR -> EXPR [13:16]
+        |       |           `--NUM_INT -> 0 [13:16]
+        |       |--SEMI -> ; [13:17]
+        |       |--LITERAL_WHILE -> while [14:8]
+        |       |   |--LPAREN -> ( [14:14]
+        |       |   |--EXPR -> EXPR [14:17]
+        |       |   |   `--GE -> >= [14:17]
+        |       |   |       |--IDENT -> i [14:15]
+        |       |   |       `--NUM_INT -> 0 [14:20]
+        |       |   |--RPAREN -> ) [14:21]
+        |       |   `--SLIST -> { [14:23]
+        |       |       |--LITERAL_SWITCH -> switch [15:12]
+        |       |       |   |--LPAREN -> ( [15:19]
+        |       |       |   |--EXPR -> EXPR [15:20]
+        |       |       |   |   `--IDENT -> i [15:20]
+        |       |       |   |--RPAREN -> ) [15:21]
+        |       |       |   |--LCURLY -> { [15:23]
+        |       |       |   |--CASE_GROUP -> CASE_GROUP [16:16]
+        |       |       |   |   |--LITERAL_CASE -> case [16:16]
+        |       |       |   |   |   |--EXPR -> EXPR [16:21]
+        |       |       |   |   |   |   `--NUM_INT -> 1 [16:21]
+        |       |       |   |   |   `--COLON -> : [16:22]
+        |       |       |   |   |--LITERAL_CASE -> case [17:16]
+        |       |       |   |   |   |--EXPR -> EXPR [17:21]
+        |       |       |   |   |   |   `--NUM_INT -> 2 [17:21]
+        |       |       |   |   |   `--COLON -> : [17:22]
+        |       |       |   |   `--SLIST -> SLIST [18:21]
+        |       |       |   |       |--EXPR -> EXPR [18:21]
+        |       |       |   |       |   `--POST_INC -> ++ [18:21]
+        |       |       |   |       |       `--IDENT -> i [18:20]
+        |       |       |   |       `--SEMI -> ; [18:23]
+        |       |       |   |--CASE_GROUP -> CASE_GROUP [19:16]
+        |       |       |   |   |--LITERAL_CASE -> case [19:16]
+        |       |       |   |   |   |--EXPR -> EXPR [19:21]
+        |       |       |   |   |   |   `--NUM_INT -> 3 [19:21]
+        |       |       |   |   |   `--COLON -> : [19:22]
+        |       |       |   |   `--SLIST -> SLIST [20:21]
+        |       |       |   |       |--EXPR -> EXPR [20:21]
+        |       |       |   |       |   `--POST_INC -> ++ [20:21]
+        |       |       |   |       |       |--SINGLE_LINE_COMMENT -> // [19:24]
+        |       |       |   |       |       |   `--COMMENT_CONTENT ->  Violation: Fall
+ through from previous branch.\r\n [19:26]
```

Comment:
try to minimize Test.java as much to avoid this line wraps.

if still required please do like you did at:
```
    |   |--COMMENT_CONTENT -> *\r\n * This is a sample Test
                                      class demonstrating Checkstyle functionality.\r\n  [1:2]
```
with indentaiton.